assets/styles/style.css

```diff
@@ -706,3 +723,149 @@ footer {
 }
 
 /* ====== CONTACT FORM END ==== */
+
+/* ====== CMODAL POPUP START ==== */
+.closeModal {
+  display: none;
+}
+
+.showModal {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  padding: 2em 1em;
+  background: #97949371;
+  backdrop-filter: blur(14px);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] You've done a fantastic job with the design in the desktop version! I have a small suggestion to make it look even better. Could you kindly align your buttons to the left, similar to the layout in the Figma template? This adjustment will bring your project closer to the intended design. 🙌

- [ ] Please also consider imcrementing a bit the width of your popup.

   ### Your design:
   ![image](https://github.com/Errytagedesign/microverse-portfolio/assets/107952724/9f9365a3-5a90-48f9-ba4e-75d0b5272353)

   ### Figma design:
   ![image](https://github.com/Errytagedesign/microverse-portfolio/assets/107952724/f76baf3f-0c85-4cfd-b237-fc6ca134a5dc)